en/developer_guide/install.rst

```diff
@@ -110,11 +110,12 @@ Launch the grunt command to generate all assets
 Load the fixtures
 -----------------
 
-Open Orchestra comes with a set of fixtures for a quick demo tour
+Open Orchestra needs some fixtures to work.
```

Comment:
I think you should talk of a production environment here. And can you give some examples to give the reader an idea of what is discussed here ?